src/types/onyx/ReimbursementAccountDraft.ts

```diff
@@ -18,7 +18,7 @@ type CompanyStepProps = {
     website?: string;
     companyTaxID?: string;
     incorporationType?: string;
-    incorporationDate?: string | Date;
+    incorporationDate?: string;
```

Comment:
Why it was updated?

Reply:
because of the typings for `DatePicker` which will return `string` and not `Date`